pom.xml

```
@@ -645,6 +645,20 @@
                         <artifactId>exec-maven-plugin</artifactId>
                         <version>${exec-maven-plugin.version}</version>
                         <executions>
+                            <execution>
+                                <id>build-gradle-library</id>
+                                <phase>compile</phase>
+                                <goals>
+                                    <goal>exec</goal>
+                                </goals>
+                                <configuration>
+                                    <workingDirectory>${project.basedir}/gradle-library</workingDirectory>
+                                    <executable>./gradlew</executable>
+                                    <arguments>
+                                        <argument>build</argument>
+                                    </arguments>
+                                </configuration>
+                            </execution>
```

I know there aren't tests yet, but in case there are one day, we should add the test phase as well.

```suggestion
                            </execution>
                            <execution>
                                <id>test-gradle-library</id>
                                <phase>test</phase>
                                <goals>
                                    <goal>exec</goal>
                                </goals>
                                <configuration>
                                    <workingDirectory>${project.basedir}/gradle-library</workingDirectory>
                                    <executable>./gradlew</executable>
                                    <arguments>
                                        <argument>test</argument>
                                    </arguments>
                                </configuration>
                            </execution>
```